packages/pn-personagiuridica-webapp/src/components/Contacts/SpecialContactElem.tsx

```diff
@@ -145,7 +145,8 @@ const SpecialContactElem = memo(({ address, recipientId }: Props) => {
             contactType={f.contactType}
             fields={[
               {
-                id: `value-${f.id}`,
+                id: `specialContacts-${address.senderId}-${f.contactType}`,
+                key: 'specialContactValue',
```

Comment:
Not use static key. Use `specialContactValue-${f.id}`

Reply:
done